src/plugins/console/public/application/models/legacy_core_editor/mode/input_highlight_rules.js

```diff
@@ -88,9 +88,35 @@ export function InputHighlightRules() {
       addEOL(['url.param'], /([^&=]+)/, 'start-sql'),
       addEOL(['url.amp'], /(&)/, 'start-sql')
     ),
+    comments: [
```

Comment:
I'm a noob when it comes to Ace's highlighting rule syntax. Are there any comments you could leave here to describe what this code does, for a noob like me? 😅